variables.tf

```diff
@@ -16,7 +16,6 @@ variable "integration_name" {
 variable "service_account_name" {
   type = string
   description = "The GCP service account name."
-  default = "sa-for-uptycs"
```

Comment:
For default inputs we should set default value in varaiables. i.e integration_name, service_account_name

Reply:
We can rely on the README to explain it.
Since we are asking customer to copy-paste the sample .tf that is pre-filled.. we can list all defaults at one place. At README.md file. If I have to maintain one here and one in README.md .. it is getting diverged.